kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/RecordEncryptionMetrics.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.filter.encryption;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Meter;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import static io.micrometer.core.instrument.Metrics.globalRegistry;
+
+public class RecordEncryptionMetrics {
+
+    public static final String VIRTUAL_CLUSTER_LABEL = "virtual_cluster";
+    public static final String TOPIC_NAME = "topic_name";
+
+    // Base Metric Names
+    private static final String RECORD_ENCRYPTION_ENCRYPTED_RECORDS = "kroxylicious_record_encryption_encrypted_records";
+    private static final String RECORD_ENCRYPTION_PLAIN_RECORDS = "kroxylicious_record_encryption_plain_records";
```

Comment:
I think we should be pushing these metrics into a separate namespace from the core proxy runtime metrics. So 
```suggestion
    private static final String RECORD_ENCRYPTION_ENCRYPTED_RECORDS = "kroxylicious_filter_record_encryption_encrypted_records";
    private static final String RECORD_ENCRYPTION_PLAIN_RECORDS = "kroxylicious_filter_record_encryption_plain_records";
```

I'm a little wary of any filter using the `kroxylicious` prefix but I guess this is a project built and supported filter so its probably appropraite.